system-addon/content-src/components/Sections/_Sections.scss

```diff
@@ -0,0 +1,76 @@
+.sections-list {
+
+  .section-top-bar {
+    position: relative;
+
+    .section-title {
+      float: left;
+    }
+
+    .section-info-option {
+      float: right;
+    }
+
+    .info-option-icon {
+      background-image: url('assets/glyph-info-option-12.svg');
+      background-size: 12px 12px;
+      background-repeat: no-repeat;
+      background-position: center;
+      height: 16px;
+      width: 16px;
+      display: inline-block;
+    }
+
+    .section-info-option div {
+      visibility: hidden;
+      opacity: 0;
+      transition: visibility 0.5s, opacity 0.5s ease-in-out;
```

Comment:
Can you make this 0.2s and add a delay of maybe 0.5s on the transition out only?